### PR TITLE
another post from the cruising team for this week - this one less lyrical

### PR DESCRIPTION
**Pull request by vlahm**

### CODE REVIEW

File: _posts/2017-07-03-cruisingUpdate1.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title:  "Getting acquainted with our city"
+date:   2017-07-03
+author: Cruising Team
+---
+
+One of the pitfalls of data science is that there's often little incentive to acquire domain knowledge. Systems under study can be easily reduced to abstractions and shoved into models that may or may not acccount for relevant details. For data science aimed at effecting social good, the inclusion of such detail is of particular importance, as social systems are often intricate and nuanced, such that modification of any one structure impacts other connected parts of the system. So it is with transportation management, and so it was that the Cruising team took to the city, to meet with stakeholders in our project and to scope out the neighborhoods we'll be studying this summer.
```
Comment (valentina-s) [inline _posts/2017-07-03-cruisingUpdate1.md:8]: effecting -> affecting

**Comment by vlahm** [comment]
I'm using the less common, verb form of "effecting" here. In this context it means "producing as an effect." http://www.dictionary.com/browse/effecting

**Comment by valentina-s** [comment]
@vlahm , I see there are several other files modified in this PR (the posts related to the Data Walk). Are you intending to update those too? In general it is good to submit unrelated changes in separate PRs. I suspect merging will add the police image which was removed by Maya. I think if you delete it on your branch and commit the change it will be ok.

**Comment by vlahm** [comment]
Hi Valentina,
Is it possible to open another pull request before the last one is closed? the reason there are so many commits in here is that i've had the same pull request open the whole time, and each new commit just gets tacked on to it. Anyway, I removed the police photo as you suggested, so I hope this works.

**Comment by valentina-s** [comment]
If you also remove the image from your folder and commit the change it will work. I don't think you need a new pull request.

**Pull request merged**